app/views/pages/start.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <h1 class="heading-large"><%= t('.pay_your_fees') %></h1>
+    <p class="lede">
+      <%= t('.leader') %>
+    </p>
+
+    <p>
+      <%= t('.use_this_service_to') %>
+    </p>
+    <ul class="list list-bullet">
+      <%= t('.list_of_uses_html') %>
+    </ul>
+    <p>
+      <%= t('.how_long_does_it_take') %>
+    </p>
+    <p>
+      <%= link_to t('.start_now') , new_case_request_path, class: 'button button-start' %>
+    </p>
+    <h2 class="heading-medium"><%= t('.before_you_start') %></h2>
+    <p>
+      <%= t('.what_you_need') %>
+    </p>
+  </div>
+
+  <div class="column-one-third">
+    <aside class="govuk-related-items" role="complementary">
```

Comment:
I just did a quick search on the repo and they do not appear to have added it, yet.  I'll leave this in for now.